src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -31,41 +36,48 @@
 from gsy_e.models.strategy.finite_power_plant import FinitePowerPlant # NOQA
 
 from gsy_e.models.leaves import Leaf # NOQA
-from gsy_e.models.leaves import *  # NOQA
-from gsy_framework.utils import convert_pendulum_to_str_in_dict, key_in_dict_and_not_none
+from gsy_e.models.leaves import *  # NOQA  # pylint: disable=wildcard-import
```

Comment:
Nit, this pylint warning is defined twice in this module.
Regardless, It would be great if we can get rid of this wildcard import here, or at least encapsulate it
```import gsy_e.models.leaves as leaves```